index.d.ts

```diff
@@ -512,6 +512,11 @@ declare module 'snowflake-sdk' {
          *  The option to pass passcode from DUO.
          */
         passcode?: string;
+
+        /**
+         *  The option to override the environment proxy value (HTTPS_PROXY) if the connection proxy is configured in GCS
+         */
+        overrideEnvProxy?: string;
```

Comment:
probably i'm missing something but since we anyways [validate whether the configured value is a boolean](https://github.com/snowflakedb/snowflake-connector-nodejs/pull/982/files#diff-549a4dcefafa5648ba3e29b9b56eb7fb0de5ee9ef37f56344f30d929d04f3319R87) , i don't understand why we expecting a string here

Reply:
My bad. It must be the boolean type.